core/src/main/java/io/questdb/cairo/wal/seq/SequencerMetadataService.java

```diff
@@ -46,7 +46,8 @@ public void addColumn(
             boolean symbolCacheFlag,
             boolean isIndexed,
             int indexValueBlockCapacity,
-            boolean isSequential
+            boolean isSequential,
+            boolean isDedupKey
```

Comment:
name

Reply:
declaring column as part of dedupe key when adding column is not a supported workflow. Lets remove this parameter

Reply:
removed parameter in https://github.com/questdb/questdb/pull/3524/commits/8699ee84624f26e3c01491745fc4ef3fd3e90633